server/services/flood.js

```diff
@@ -64,6 +66,7 @@ module.exports = {
     return util.getJson(`${serviceUrl}/station/${id}/${direction}`)
   },
 
+  // get stations within the binding box defined using coordinates x1, y1 & x2, y2
```

Comment:
Is it worth using joi to add parameter validation to document the assumptions inherent in the code rather than adding comments? (e.g. validate bbox is an array of 4 numbers).

Maybe even map bbox to a coordinate pair object ( e.g. `{ topLeft: { lat: ..., lon: ... }, bottomRight: { lat: ..., lon: ... } }`) to make the code intent more explict rather than use comments.